warehouse/email/ses/models.py

```diff
@@ -230,18 +233,17 @@ def _get_email(self):
 class EmailMessage(db.Model):
     __tablename__ = "ses_emails"
 
-    created = mapped_column(DateTime, nullable=False, server_default=sql.func.now())
-    status = mapped_column(
+    created: Mapped[datetime] = mapped_column(server_default=sql.func.now())
+    status: Mapped[Enum] = mapped_column(
         Enum(EmailStatuses, values_callable=lambda x: [e.value for e in x]),
-        nullable=False,
         server_default=EmailStatuses.Accepted.value,
     )
 
-    message_id = mapped_column(Text, nullable=False, unique=True, index=True)
-    from_ = mapped_column("from", Text, nullable=False)
-    to = mapped_column(Text, nullable=False, index=True)
-    subject = mapped_column(Text, nullable=False)
-    missing = mapped_column(Boolean, nullable=False, server_default=sql.false())
+    message_id: Mapped[str] = mapped_column(unique=True, index=True)
+    from_: Mapped[str] = mapped_column("from")
+    to: Mapped[str] = mapped_column(index=True)
+    subject: Mapped[str]
+    missing: Mapped[bool] = mapped_column(server_default=sql.false())
```

Comment:
Need to import it, but:

```suggestion
    missing: Mapped[bool_false]
```

Reply:
https://github.com/pypi/warehouse/pull/14270/commits/1b48842f7b91ca352b1eb351d110723d669db5cb